react/MuiCozyTheme/theme.js

```diff
@@ -258,7 +259,61 @@ theme.overrides = {
 }
 
 // Override default shadow for elevation 8 used for Popover
-theme.shadows[8] =
+normalTheme.shadows[8] =
   '0rem 0.125rem 0.375rem 0rem rgba(50, 54, 63, .19), 0rem 0.375rem 1.25rem 0rem rgba(50, 54, 63, .19)'
 
-export default theme
+export const invertedTheme = {
+  ...normalTheme,
+  palette: {
+    ...normalTheme.palette,
+    type: 'dark',
```

Comment:
Nice we were discussing this with Joel, I had the feeling that the dark color was a "coincidence" for the inverted palette (you could have a normal palette that was dark, and its inverted would be light) and that the theme should contain this information (see https://github.com/cozy/cozy-ui/pull/1316, and my awkward `useDarkBackground`.

I am glad that material UI support this 🎉